packages/core/contracts/oracle/implementation/SpamGuardIdentifierLib.sol

```diff
@@ -1,3 +1,6 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
```

Comment:
```suggestion
pragma solidity 0.8.15;
```
Locked solidity version to `0.8.15` [here](https://github.com/UMAprotocol/protocol/pull/4075)